### PR TITLE
api/firmware: remove BitBoxBase and RandomNumber

### PR DESCRIPTION
**Pull request by benma**
Copied updated .proto files from
https://github.com/digitalbitbox/bitbox02-firmware/tree/firmware/v9.7.0/messages.

### CODE REVIEW

File: api/firmware/messages/hww.proto [hunk 1]
```diff
@@ -19,12 +19,10 @@ import "common.proto";
 
 import "backup_commands.proto";
 import "bitbox02_system.proto";
-import "bitboxbase.proto";
```
Comment (x1ddos) [inline api/firmware/messages/hww.proto:22]: `api/firmare/messages/bitboxbase.*` files are still there

File: api/firmware/messages/hww.proto [hunk 2]
```diff
@@ -19,12 +19,10 @@ import "common.proto";
...
 import "btc.proto";
 import "eth.proto";
 import "keystore.proto";
 import "mnemonic.proto";
-import "random_number.proto";
```
Comment (x1ddos) [inline api/firmware/messages/hww.proto:27]: `api/firmare/messages/random_number.*` files are still there

**Review by x1ddos — COMMENTED** [review]
"Copied ... from bitbox02-firmware/tree/firmware/v9.7.0/messages" - did you mean from bitbox02-firmware/commit/0c3cd91 since that's where bitboxbase protos were removed?

In general, I already forgot: how to do it? For example, I see [backup.proto](https://github.com/digitalbitbox/bitbox02-firmware/blob/a273a88c/messages/backup.proto) in the firmware repo but not here and I can't remember why. What's the command to sync proto files? Would be very nice to have it in the readme.

File: api/firmware/messages/hww.proto (hunk 1 above)
Comment (benma) [reply api/firmware/messages/hww.proto:22]: done

File: api/firmware/messages/hww.proto (hunk 2 above)
Comment (benma) [reply api/firmware/messages/hww.proto:27]: done

**Comment by benma** [comment]
@x1ddos added README instructions on how to update the files. PTAL

**Review by x1ddos — APPROVED** [review]
LGTM. Thanks! :pray:

**Pull request merged**